py3status/module.py

```diff
@@ -657,6 +657,16 @@ def deprecation_log(item):
             if not hasattr(self.module_class, "py3"):
                 setattr(self.module_class, "py3", Py3(self))
 
+            # Subscribe to udev events if on_udev_* dynamic variables are
+            # configured on the module
+            on_udev_vars = [
+                i for i in dir(self.module_class) if i.startswith("on_udev_")
+            ]
+            for on_udev_var in on_udev_vars:
```

Comment:
(?) This looks a bit weird. Maybe simplify it a bit more if you want it.
```python
    for param in dir(self.module_class):
        if param.startswith("on_udev_"):
            trigger_action = getattr(self.module_class, param)
            self.add_udev_trigger(trigger_action, param[8:])
```

Reply:
I have to agree with @lasers here.  I think his version is more clean and readable.